test/test_unit.jl

```diff
@@ -615,6 +616,16 @@ end
     @test_nowarn show(stdout, time_series)
     @test_throws ArgumentError TimeSeriesCallback(semi, [(1.0, 1.0)]; interval = -1)
     @test_throws ArgumentError TimeSeriesCallback(semi, [1.0 1.0 1.0; 2.0 2.0 2.0])
+    # Test the 2D unstructured mesh version of the callback
+    @test_nowarn_mod trixi_include(@__MODULE__,
+                                   joinpath(examples_dir(), "unstructured_2d_dgsem",
+                                            "elixir_euler_time_series.jl"),
+                                   tspan = (0.0, 0.2))
+
+    point_data_1 = time_series.affect!.point_data[1]
+    @test all(isapprox.(point_data_1[1:4],
+                        [1.9546882708551676, 1.9547149531788077,
+                            1.9547142161310154, 3.821066781119142]))
```

Comment:
Since we already test this elixir in `test_unstructured_2d.jl` wouldn't it make sense to just append this test there instead of running it twice?

I will also add an extension of the `TimeSeriesCallback` for TreeMesh1D and 3D so this would introduce a few additional full elixir tests into the unit testing.

Reply:
Yes, this could be done. I would just need to adjust the `interval` in the new elixir to use 1 rather than 10. Otherwise, when we run the tests with coverage (where only one timestep is taken) there is no time series data to test against.

Reply:
There are `coverage_override` options (or something with a similar name) to change, e.g., `maxiters` when coverage is enabled.

Reply:
Right, I forgot that existed. I can alter the testing then and remove the redundant one.